api/envoy/service/load_stats/v2alpha/orca.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package envoy.service.load_stats.v2alpha;
+
+option java_outer_classname = "OpenRCAProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.service.load_stats.v2alpha";
+option go_package = "v2alpha";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// Out-of-band (OOB) load reporting service for frontends to periodically get
+// load information from backends.
+service OpenRCAService {
+  rpc StreamCoreMetrics(stream LoadReportRequest) returns (stream LoadReport);
+}
+
+message LoadReportRequest {
+  // Interval for generating Open RCA core metric responses.
+  google.protobuf.Duration report_interval = 1;
+  // Request costs to collect. If this is empty, all known requests costs tracked by
+  // the load reporting agent will be returned. This provides an opportunity for
+  // the client to selectively obtain a subset of tracked costs.
+  repeated string request_cost_names = 2;
+}
+
+message LoadReport {
```

Comment:
Nit: I think maybe rename to `OrcaLoadReport` to avoid confusion with LRS. Same with `OrcaLoadReportRequest`.